### ISSUE TITLE
OPTIONAL System's Context Menu

### ISSUE THREAD

**Issue by anonymous1184** [fman-community/fman#775]
### TL;DR:

Mimic 7zip's implementation of System's Context Menu as sub-menu.

![Screenshot 20200919210022](https://user-images.githubusercontent.com/53758552/93692646-62b3cb80-fabb-11ea-94f7-7ac5785ee20d.png)

### Long version:

I'm a really long time user of Sublime Text Editor because it fits my keyboard-centric usage of computers. With fman I found the Holy Grail, thanks you so much for it and honestly is a shame there's not enough buzz around it (I found it less than a week ago).

Now, since I need to use macOS with W10 in Seamless Mode of Virtual Box and a miriad of VMs with Linux, having a standard File Manager across OS' is trilling. However for Windows a little more extensibility than the one customized context menu and plugins is needed; precisely the one provided by the OS for each file type.

(And yes, I've searched and read context menu related issues like #466, #678, the [blog](https://fman.io/blog/fmans-new-context-menu/) and the [help](https://fman.io/docs/context-menu).)

7-zip File Manager in Windows (as shown in the image) has the perfect solution to the problem (having the system's context menu as a sub-menu of its own). It can be toggled easily for the users who don't need/want it.

What you think? This will put to rest current requests and please both: users who are happy with the current context menu and user who want system's.

Stay safe!

**Comment by gerhard-h** [comment]
I really like fman for its speed and its ctl+sft+p feature. 
But being able to use OS context menu in an alternativ file manager is essential for me.
